docs/observability/README.md

```diff
@@ -0,0 +1,20 @@
+# KFServing Observability
+
+The [Knative Observability](https://github.com/knative/observability) project is deprecated and no longer ships releases. So for the time being, it is necessary to look deeper for metrics regarding deployed `InferenceServices`.
```

Comment:
Should we reference [Istio grafana dashboards](https://istio.io/latest/docs/ops/integrations/grafana/) ?